keymap/vim.js

```diff
@@ -940,6 +942,25 @@
       }
     };
 
+    /**
+     * Defines an external register.
+     *
+     * The name should be a single character that will be used to reference the register.
+     * The register should support setText, pushText, clear, and toString(). See Register
+     * for a reference implementation.
+     */
+    function defineRegister(name, register) {
+      var registers = vimGlobalState.registerController.registers[name];
```

Comment:
Shouldn't it be `var registers = vimGlobalState.registerController.registers[name];`? I can't define a register because it throws "Can't read property <my_register_name> of underfined" later in the code (957 line).